reforis-angelcam-connector-plugin/Makefile

```
@@ -4,7 +4,11 @@ PKG_NAME:=reforis-angelcam-connector-plugin
 PKG_VERSION:=0.1.0
 PKG_RELEASE:=1
 
-PKG_LICENSE:=Apache-2.0
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/angelcam/reforis-angelcam-connector-plugin/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=c047c81b1150185bc305c19b00a1fa976f0db3b4b8f66f11d376419b12a70aac
+
+PKG_LICENSE:=MIT
 
 PKG_BUILD_DEPENDS:=\
```

Just nitpicking. You could not know about this variable.

```suggestion
PKG_LICENSE:=MIT
PKG_LICENSE_FILES:=LICENSE

PKG_BUILD_DEPENDS:=\
```